.github/BOTMETA.yml

```diff
@@ -449,7 +449,7 @@ files:
     ignore: resmo
     maintainers: dmtrs
   $modules/consul:
-    ignore: colin-nolan Hakon
+    ignore: colin-nolan Hakon Ilgmi
```

Comment:
If you contribute a module, you should also add yourself as a maintainer, and not use `ignore`.